LathBotFront/Commands/ReactionCommands.cs

```diff
@@ -114,6 +118,8 @@ public async Task Pat(CommandContext ctx, [Description("Who to pat")] DiscordMem
                 await ctx.RespondAsync(new DiscordMessageBuilder().WithStickers([ctx.Guild.Stickers[1013888253272801310]]));
             else if (member.Id == 387325006176059394)//julian
                 await ctx.RespondAsync($"Julian!, you just got pat by {ctx.Member.Mention}");
+            else if (member.Id == 312936081819697154)//Ryguy6001
+                await ctx.RespondAsync($"Ryguy doesn't like getting patted.");
```

Comment:
Overlooked that earlier sorry, but same for you as with tophat, noone gets to not get pat, just go yelling at people that try to do so